content/en/apps/examples/covid-rdt-reference-app.md

```diff
@@ -0,0 +1,490 @@
+
+* contact:clinic:create.xml
+* contact:clinic:edit.xml
+* contact:district_hospital:create.xml
+* contact:district_hospital:edit.xml
+* contact:health_center:create.xml
+* contact:health_center:edit.xml
+* contact:person:create.xml
+* contact:person:edit.xml
+* covid19_rdt_capture.xml
+* covid19_rdt_provision.xml
+
+The last two forms can be used to query [the reports API ]({{< ref "apps/reference/api#get-apiv2exportreports" >}}) to get RDT provions and captures.  To get all the capture reports into a file called `output.csv`, you would use this call:
+
+```shell
+curl "http://LOGIN:PASSWORD@HOSTNAME/api/v2/export/reports?filters[search]=&filters[forms][selected][0][code]=covid19_rdt_capture" > output.csv
+```
+
+### Base64 image extraction
+
+To extract the binary image from the ASCII value in a report, you can view it in the CHT or you use a script to do so.  Here's an example of using Bash utilities (`sed`, `cut`, `tr` and `base64`) in Linux to get the image in the 1st row of data after the column headers into the file `image.jpg`. 
+
+```shell
+sed '2q;d' output.csv | cut -d',' -f15 | tr -d '"' | base64 -d > image.jpg
+```
+
+When retrieving [JSON](#capture-1), this value is found in `capture.android-app-outputs.rdt_session_bundle.rdt_session_result_bundle.rdt_session_result_extra_images.cropped` field.
+
+{{% alert title="Note" %}}The reports API always outputs in CSV, but the COVID-19 application uses [Base64 encoding](https://en.wikipedia.org/wiki/Base64) to store the images as text as noted above.  These may misbehave when opened them in a spreadsheet application like LibreOffice ("maximum number of characters per cell exceeded") or Excel (silently clipped to 32k chars) as they're thousands, if not hundreds of thousands, of characters long.  Be sure to programmatically process these into image files as needed. {{% /alert %}}
+
+
+## Directory structure
+
+Shown below is a filtered file structure of the COVID-19 app where you'll want to focus your customization efforts:
+
+```
+├── forms
+│   ├── app
+│   │   ├── covid19_rdt_capture.properties.json
+│   │   ├── covid19_rdt_capture.xlsx
+│   │   ├── covid19_rdt_capture.xml
+│   │   ├── covid19_rdt_provision.properties.json
+│   │   ├── covid19_rdt_provision.xlsx
+│   │   └── covid19_rdt_provision.xml
+├── tasks.js
+```
+
+The `forms/app/covid19_rdt_provision` and `forms/app/covid19_rdt_capture` forms (`xlsx`, `xml` and `properties.json`) represent the provision and capture portions of the forms.  The tasks that get created are defined in `tasks.js`.  Not shown are standard contact definitions in `forms/contact/*` as well as supporting configurations for icons and other CHT application settings.  
+
+To read more about how these files all work together, see [app forms]({{< ref "apps/reference/forms/app" >}}), [contact forms]({{< ref "apps/reference/forms/contact" >}}), and [task]({{< ref "apps/reference/tasks" >}}) reference documentation
+
+## JSON Reference 
```

Comment:
It's nice to include the JSON of reports as reference. Given the "JSON Reference" header though, I was expecting JSON definition with descriptions, rather than a sample -- but that may just be me! Assuming it's not, then it may be nice to clarify the contents of this section in the header.

```suggestion
## Example form submission
```